gitlab/const.py

```diff
@@ -27,17 +30,48 @@
 MAINTAINER_ACCESS: int = 40
 OWNER_ACCESS: int = 50
 
+
+class AccessLevel(Enum):
+    NO_ACCESS: int = 0
+    MINIMAL_ACCESS: int = 5
+    GUEST_ACCESS: int = 10
+    REPORTER_ACCESS: int = 20
+    DEVELOPER_ACCESS: int = 30
+    MAINTAINER_ACCESS: int = 40
+    OWNER_ACCESS: int = 50
+
+
+# TODO: deprecate these in favor of Visibility
 VISIBILITY_PRIVATE: str = "private"
 VISIBILITY_INTERNAL: str = "internal"
 VISIBILITY_PUBLIC: str = "public"
 
+
+class Visibility(Enum):
+    VISIBILITY_PRIVATE: str = "private"
+    VISIBILITY_INTERNAL: str = "internal"
+    VISIBILITY_PUBLIC: str = "public"
```

Comment:
Same as above, see https://gitlab.com/gitlab-org/gitlab/-/blob/e97357824bedf007e75f8782259fe07435b64fbb/lib/gitlab/visibility_level.rb#L23-25

```suggestion
class Visibility(Enum):
    PRIVATE: str = "private"
    INTERNAL: str = "internal"
    PUBLIC: str = "public"
```